design/ShaderLanguageFAQ.md

```diff
@@ -130,28 +135,24 @@ for handling WGSL, including parsing, validation, and compilation.
 
 At this time, we expect at least three essentially independent
 implementations of WGSL ingestion as part of browser support for WebGPU.
-Independent implementation serves as an important check on the quality
-of the WGSL and WebGPU specifications.
+We expect the existence of these independent implementations to lead to a
+community and culture of competition and innovation, rather than stagnation.
 
 
 ## What about my existing shaders, compilers, and other tooling?
 
 This is also expressed as “the last thing I need is another shader language”.
 (See https://xkcd.com/927/)
 
-For developers invested in the SPIR-V ecosystem for Vulkan and OpenGL,
-the key will be convertibility between WGSL and SPIR-V.
-For shaders that only use WebGPU features, WebGPU Community Group
-members expect to develop tooling to faithfully convert between WGSL
-and “Shader” SPIR-V.
-By “faithful”, we mean that all salient properties of a shader should
-be preserved by a round trip conversion between the two ways of writing
-the shader: e.g. it should retain its semantics and its performance
-characteristics.
-Furthermore, those WebGPU Community Group members plan to add an option
-to existing mainstream SPIR-V compilers to emit WGSL directly.
-We expect other SPIR-V tooling (e.g. fuzzers, optimizers) can be reused,
-via the addition of a conversion step between WGSL and SPIR-V at their
-I/O boundaries.
-
-
+WGSL is a new language. Over time, we expect a collection of compilers will
```

Comment:
This paragraph looks to be downplaying the role of SPIR-V in the picture. I think it's most important for the readers of this FAQ to come out with the information that their SPIR-V based shader toolchains will be compatible with WGSL.

I mean, the FAQ is meant to reduce the confusion, not make the readers ask more questions. In simple wording what I'd expect it to say is "WGSL is based on SPIR-V semantics, and we guarantee that the existing SPIR-V pipelines will be able to target WGSL eventually, if they consider the reduced subset of SPIR-V semantics available".

Reply:
Agree with @kvark  here.   This text was specifically was aimed at the SPIR-V community who were, uh, hungry for information.

Reply:
Evidence that my earliest draft of the FAQ satisfied an information need in the SPIR-V community:  [before](https://twitter.com/aras_p/status/1232989572306526212) and [after](https://twitter.com/aras_p/status/1233054419761160194)